webui/src/docs/rest-api.jsx

```diff
@@ -1,244 +1,540 @@
+                "self": "https://vm0045.kaj.pouta.csc.fi/api/records/?sort=mostrecent&q=&page=1"
+              }
+            }}
+            </Returns>
+        </Example>
+
+        <h3 id="list-records-per-community">List records per community</h3>
+        <p>List all records of a specific community.</p>
+        <ul>
+            <li><p>URL path: /api/records/?q=community:COMMUNITY_ID</p></li>
+            <li><p>HTTP method: GET</p></li>
+            <li><p>Required parameters: access_token</p></li>
+            <li><p>Returns: the list of records (in JSON format) or an error message.</p></li>
+        </ul>
+        <Example>
+            curl https://$HOSTNAME/api/records/?q=community:$COMMUNITY_ID?access_token=$ACCESS_TOKEN
+            <Returns>
+            {{
+              "aggregations": {
+                "type": {
+                  "buckets": [],
+                  "doc_count_error_upper_bound": 0,
+                  "sum_other_doc_count": 0
+                }
+              },
+              "hits": {
+                "hits": [
+                  {
+                    "created": "2016-10-24T11:29:27.016892+00:00",
+                    "id": "f7fddf6f111f4362a9e4661294e2b59e",
+                    "links": {
+                      "files": "https://vm0045.kaj.pouta.csc.fi/api/files/90ea3483-2792-4483-9392-7d624b610398",
+                      "self": "https://vm0045.kaj.pouta.csc.fi/api/records/f7fddf6f111f4362a9e4661294e2b59e"
+                    },
+                    "updated": "2016-10-24T11:29:27.016900+00:00",
+                    "..." : "..."
+                  },
+                  "..."
+                ],
+                "total": 32
+              },
+              "links": {
+                "next": "https://vm0045.kaj.pouta.csc.fi/api/records/?sort=bestmatch&q=community%3Ae9b9792e-79fb-4b07-b6b4-b9c2bd06d095&size=10&page=2",
+                "self": "https://vm0045.kaj.pouta.csc.fi/api/records/?sort=bestmatch&q=community%3Ae9b9792e-79fb-4b07-b6b4-b9c2bd06d095&size=10&page=1"
+              }
+            }}
+            </Returns>
+        </Example>
+
+        <h3>Search records</h3>
+        <p>Search all the published records for a query string.</p>
+        <ul>
+            <li><p>URL path: /api/records/?q=$QUERY_STRING</p></li>
+            <li><p>HTTP method: GET</p></li>
+            <li><p>Required parameters: access_token</p></li>
+            <li><p>Returns: the list of matching records (in JSON format) or an error message.</p></li>
+        </ul>
+        <Example>
+            curl https://$HOSTNAME/api/records/?q=$QUERY_STRING?access_token=$ACCESS_TOKEN
+        </Example>
+
+        <h3>Search drafts</h3>
+        <p>Search for all drafts (unpublished records) that are accessible by the requestor. Usually this means own records only.</p>
+        <ul>
+            <li><p>URL path: /api/records/?drafts=1</p></li>
```

Comment:
normally there is no need to add the `=1` in `?drafts=1`. If it doesn't work without it it's a bug.